.github/workflows/create-github-release.yml

```diff
@@ -0,0 +1,53 @@
+name: Create GitHub release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: 'Version to publish'
+        required: true
+      title:
+        type: string
+        description: 'Release title'
+        required: true
+      deploy-sha:
+        type: string
+        description: 'Commit SHA to create release from'
+        required: true
+  
+  workflow_call:
+    inputs:
+      version:
+        type: string
+        description: 'Version to publish'
+        required: true
+      title:
+        type: string
+        description: 'Release title'
+        required: true
+      deploy-sha:
+        type: string
+        description: 'Commit SHA to create release from'
+        required: true
+
+jobs:
+  create-github-release:
+    name: Create GitHub release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create release
+        uses: ncipollo/release-action@v1
+        with:
+          name: ${{ inputs.title }}
+          tag: ${{ inputs.version }}
```

Comment:
It appears like this produces a version like `"1.10.0"`
![image](https://github.com/Constructor-io/constructorio-ui-autocomplete/assets/2954126/a18343c6-3728-46ce-a075-08d9a0c5c94a)
We usually do `v1.10.0`
![image](https://github.com/Constructor-io/constructorio-ui-autocomplete/assets/2954126/98a5852b-8b58-4d6e-b4d2-78a619a91108)

Can we strip the `"` and add a `v` in front? Alternatively, is there a way to infer this from package.json?

Reply:
I think I found the answer to my question. It looks like value gets pushed to this from the previous step. Can we still add the `v` and get rid of `"`"

Reply:
Great catch!

Reply:
Made this adjustment with the latest commit!